website/content/docs/secrets/databases/elasticdb.mdx

```diff
@@ -130,6 +130,16 @@ Now, Elasticsearch is configured and ready to be used with Vault.
        client_key=$ES_HOME/config/certs/elastic-certificates.key.pem
    ```
 
+## Usage
+
+After the secrets engine is configured and a user/machine has a Vault token with
+the proper permission, it can generate or rotate credentials.
+
+### Dynamic Roles
+
+Dynamic roles generate new users and credentials on demand.
+The credentials are unique for each client and created when queried.
```

Comment:
I don't understand what this is trying to say. We create new *users*? And what does "when queried" mean? Is that when the client connects?

Reply:
Yes, Vault creates new elasticsearch users every time a Vault client requests new credentials for a dynamic role by calling the `database/creds/my-role` endpoint.

Requested is proabably a better word than when queried :P

What do you mean by when a Vault client connects?